core/tests/webdriverio_utils/ExplorationPlayerPage.js

```diff
@@ -220,6 +220,13 @@ var ExplorationPlayerPage = function() {
     ).toMatch(richTextInstructions);
   };
 
+  this.waitForInvisibilityOfLessonCompletionMessage = async function() {
+    var lessonCompletionMessage = $('.e2e-test-lesson-completion-message');
+    await waitFor.invisibilityOf(
+      lessonCompletionMessage,
+      'Last conversation content took took long to disappear');
```

Comment:
I don't understand what 'last conversation content' refers to here. It sounds different from 'lesson completion message'?

Reply:
You are right should use 'lesson completion message' for maintaining consistency in the codebase. last element in conversation content list is 'lesson completion message' itself.